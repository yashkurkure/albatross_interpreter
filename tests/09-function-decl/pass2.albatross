var global int := 10;

fun function int () {
  return global+1*2;
}

return global+3;
