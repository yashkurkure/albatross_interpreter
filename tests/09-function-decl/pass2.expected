(10)
((global)+((1)*(2)))
((global)+(3))

